From 8c2b4b4a4f761d5d87fe4fab63e98f781a35f60d Mon Sep 17 00:00:00 2001
From: James Ausmus <james.ausmus@intel.com>
Date: Thu, 10 Oct 2013 22:35:00 -0700
Subject: Re-create dirty merge b599c89

Fix code up to (applicable) post-merge state after
b599c89e8c5cf0c37352e0871be240291f8ce922 ("Merge tag 'v3.12-rc2' into
drm-intel-next")

Signed-off-by: James Ausmus <james.ausmus@intel.com>
(cherry picked from
 https://chromium.googlesource.com/chromiumos/third_party/kernel-next
 chromeos-3.10, 29f638bb7b8939c1159e80a0294ff38d7eae0360)
Signed-off-by: Darren Hart <dvhart@linux.intel.com>
---
 drivers/gpu/drm/i915/intel_drv.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_drv.h b/drivers/gpu/drm/i915/intel_drv.h
index 3a7ef151f24a..4b75328462ed 100644
--- a/drivers/gpu/drm/i915/intel_drv.h
+++ b/drivers/gpu/drm/i915/intel_drv.h
@@ -826,8 +826,8 @@ extern int intel_dotclock_calculate(int link_freq,
 				    const struct intel_link_m_n *m_n);
 extern void ironlake_check_encoder_dotclock(const struct intel_crtc_config *pipe_config,
 					    int dotclock);
-extern void i915_disable_vga_mem(struct drm_device *dev);
 
 extern bool intel_crtc_active(struct drm_crtc *crtc);
+extern void i915_disable_vga_mem(struct drm_device *dev);
 
 #endif /* __INTEL_DRV_H__ */
-- 
1.8.5.rc3

